### DISCUSSION TITLE
非常希望字體轉換功能可以回歸

### DISCUSSION THREAD

**Discussion by AkiraChisaka** [tongwentang/tongwentang-extension#45] [General]
目前感覺市面上依然很難找到跟舊版同文堂一樣簡單好用的插件。

非常希望字體轉換功能可以回歸。

或者希望舊版同文堂可以回歸允許下載，作爲一個分開的Chrome插件。

**Comment by t7yang** [comment]
我在 #4 已經說明過了，可以參考。
